Source: me
Upstream: NO
Reason: Works around linkage problem(Patch might not be right.)
--- uget-1.5.0/uget-gtk/Makefile.am~	2009-12-10 16:00:20.013893746 +0800
+++ uget-1.5.0/uget-gtk/Makefile.am	2009-12-10 16:01:07.122155990 +0800
@@ -4,7 +4,7 @@
 uget_gtk_CPPFLAGS = -DDATADIR='"$(datadir)"' -I$(top_srcdir)/include -I$(top_srcdir)/uget-gtk
 uget_gtk_CFLAGS   = @LFS_CFLAGS@  @GTK_CFLAGS@ @GIO_CFLAGS@ @GTHREAD_CFLAGS@ @CURL_CFLAGS@ @LIBNOTIFY_CFLAGS@ @GSTREAMER_CFLAGS@
 uget_gtk_LDFLAGS  = @LFS_LDFLAGS@ @GTK_LIBS@   @GIO_LIBS@   @GTHREAD_LIBS@   @CURL_LIBS@   @LIBNOTIFY_LIBS@   @GSTREAMER_LIBS@
-uget_gtk_LDADD    = $(top_builddir)/src/libuglib.a
+uget_gtk_LDADD    = $(top_builddir)/src/libuglib.a $(LIBNOTIFY_LIBS) $(GSTREAMER_LIBS)
 
 uget_gtk_SOURCES  = \
 	ug_data_app.c  ug_category.c  ug_item.c \
